packages/react-ui-map/src/Mapbox/index.js

```diff
@@ -57,8 +57,23 @@ export default class Mapbox extends PureComponent {
 
       sources.forEach((v, i) => {
         map.addSource(sources[i].id, sources[i].source)
+      })
+
+      layers.forEach((v, i) => {
         map.addLayer(layers[i].layer)
       })
+
+      map.on('mouseenter', 'markers-default', () => {
+        map.getCanvas().style.cursor = 'pointer';
+      })
+
+      map.on('mouseleave', 'markers-default', () => {
+        map.getCanvas().style.cursor = '';
+      })
+
+      map.on('click', 'markers-default', (e) => {
+        map.setFilter('markers-active', ['==', 'title', e.features[0].properties.title])
```

Comment:
i would prefer to see events and interactions as props - as its currently implemented, click bindings are controlled within the component rather than outside it making this inflexible